assets/css/custom.css

```diff
@@ -424,7 +424,7 @@ body:last-child .btn-u, x:-moz-any-link {
 .browse-courses {
     display: inline-block;
     border-radius: 6px;
-    margin: 20px 0px 0px 10px;
+    margin: 20px 10px 12px;
```

Comment:
I tried to use table and table cell display attributes but the problem is that it will break other elements on other pages where the class contact-us is used.
I would need to call the elements on the blog page differently and it seems wrong